Jenkinsfile

```diff
@@ -62,29 +60,26 @@ node('docker && imgtec') {  // Only run on internal slaves as build takes a lot
             ])
 
             // Default config
-            sh "cp ${env.DEFCONFIG_FILE} .config"
+            sh "cp ${params.CONFIG_FILE?.trim()} .config"
 
             // Versioning
-            sh "sed -i 's/.*CONFIG_VERSION_NUMBER.*/CONFIG_VERSION_NUMBER=\"${env.VERSION?.trim() ?: 'j' + env.BUILD_NUMBER}\"/g' .config"
+            sh "sed -i 's/.*CONFIG_VERSION_NUMBER.*/CONFIG_VERSION_NUMBER=\"${params.VERSION?.trim() ?: 'j' + env.BUILD_NUMBER}\"/g' .config"
             sh 'scripts/getver.sh > version'
 
-            // Build tools
-            if (env.BUILD_TOOLS == 'true') {
-                echo 'Enabling toolchain, image builder and sdk creation'
-                sh 'echo \'' \
-                 + 'CONFIG_MAKE_TOOLCHAIN=y\n' \
-                 + 'CONFIG_IB=y\n' \
-                 + 'CONFIG_SDK=y\n' \
-                 + '\' >> .config'
-            }
+            // Build tools/sdks
+            echo 'Enabling toolchain, image builder and sdk creation'
```

Comment:
why do we need to generate toolchain/sdk everytime? are we making any use of that in any CI jobs? i.e. building creator kit apps using the generated SDK or ImageBuilder or so? IMHO it was good to keep BUILDS_TOOLS as optional.